remove_library.php

```diff
@@ -0,0 +1,94 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Remove h5p library.
+ *
+ * @package    mod_hvp
+ * @author     Andrew Madden <andrewmadden@catalyst-au.net>
+ * @copyright  2022 Catalyst IT
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+require(__DIR__ . '/../../config.php');
+global $CFG, $DB, $PAGE, $OUTPUT;
+require_once($CFG->libdir . '/adminlib.php');
+require_once($CFG->libdir . '/modinfolib.php');
+require_once($CFG->dirroot . '/course/lib.php');
+require_once($CFG->dirroot . '/mod/hvp/lib.php');
+
+require_admin();
+
+$options = [
+    'help' => optional_param('help', false, PARAM_BOOL),
+    'library-id' => optional_param('library-id', '', PARAM_ALPHANUMEXT),
+    'library-title' => optional_param('library-title', '', PARAM_ALPHANUMEXT),
+    'major-version' => optional_param('major-version', '', PARAM_ALPHANUMEXT),
```

Comment:
I think everywhere lets make these params a little more verbose and less painful to type, especially when doing bulk maintenance.

library-id -> id
library-title -> title
major-version -> major
minor-version -> minor
patch-version -> patch

Reply:
Also, the version input should be consistent with CLI. I prefer the --version=2.7.1, but that doesnt matter as much as long as the 2 methods are consistent

Reply:
Can't pass a period (`.`) as a query param in the url. I wanted to leave the --version=1.2.3 for the cli as it's much easier to use.

Will update the names, but leaving the versions as the full names as I don't think 'major' is descriptive enough.

Reply:
Realised I needed to use PARAM_RAW, instead, for the optional params. Both scripts now only accept `id`, `title` and `version`.